src/correctPath.js

```diff
@@ -0,0 +1,65 @@
+const isWin = process.platform === 'win32';

```

Comment:
Unixify (and the things it depends on) were needed to fix win32 support.

However, given the low-level nature of this, I opted to inline it rather than take a dependency on unixify itself. 

This reduces dependencies back to none.